src/useNextSanityImage.ts

```diff
@@ -123,6 +142,7 @@ export function useNextSanityImage(
 		}
 
 		const originalImageDimensions = getImageDimensions(id);
+		const croppedImageDimensions = getCroppedDimensions(image);
```

Comment:
It might be better to pass the `originalImageDimensions` to `getCroppedDimensions` to prevent calculating the original/base dimensions twice.